gui/default/syncthing/settings/discardChangesConfirmation.html

```diff
@@ -0,0 +1,16 @@
+<modal id="discard-changes-confirmation" status="warning" icon="fas fa-question-circle" heading="{{'Pending changes' | translate}}" large="no" closeable="yes">
+  <div class="modal-body">
+    <p style="overflow: hidden; text-overflow: ellipsis; white-space: nowrap;">
+      <span translate>You have unsaved changes. Would you like to discard them?</span>
```

Comment:
Maybe change `Would you like to discard them` to `Do you really want to discard them?`